### PR TITLE
33 issue templates

### PR DESCRIPTION
**Pull request by bms63**
Hi all, Create some templates, which I just stole from admiral. Let me know if these can work for now.

### CODE REVIEW

**Comment by bms63** [comment]
image
https://github.com/pharmaverse/admiralonco/issues/new/choose  Check these out for quick view of what they look like.  

You can also render them on github for a peak as well 
image

**Comment by kodesiba** [comment]
@bms63 any chance we want to bring over a PR template as well.  Not sure if we fully fit into the same scope as {admiral} but would be good to have something for a PR template

**Comment by bms63** [comment]
Yeah!  I'll make one up that is more for our scope but has that **pharmaversace** feel :)

**Comment by bms63** [comment]
Hi all,  Here is the pull request template 
image

**Review by kodesiba — APPROVED** [review]

**Comment by kodesiba** [comment]
@bms63 you're good to merge this in unless @thomasp-b or @nicholas-masel have anything else to add

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- [ ] The spirit of timber is met in your Pull Request
+- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
+- [ ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
+- [ ] Creation/updates to relevant roxygen headers and examples 
+- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
+- [ ] Address any updates needed for vignettes and/or templates
+- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
+- [ ] Link the issue so that it closes after successful merging. 
+- [ ] Address all merge conflicts and resolve appropriately 
+- [ ] Pat yourself on the back for a job well done!  Much love to your accomplishment!
```
Comment (thomasp-b) [inline .github/pull_request_template.md:22]: A minor detail but sometimes tasks are introduced by actions ("updated", "run", "address", etc.), sometimes they are introduced by nouns or sentences ("creation", "code is formatted (...)", etc.). Should we try to be more consistent?

**Review by nicholas-masel — APPROVED** [review]
Really cool!  I need to dig into what you've done here some more and learn this stuff.  Nothing from me here with my 0 experience on this part :)

**Pull request merged**